src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  const sale1 = 20;
+  const sale2 = 50;
+  const duration1 = 3;
+  const duration2 = 7;
+
+  if (days < duration1) {
+    return days * rent;
+  }
+
+  if (days < duration2) {
+    return days * rent - sale1;
+  } else {
+    return days * rent - sale2;
+  }
```

Comment:
```suggestion
  }
  return (days * rent - sale2);
```

you don't need the last `else`. Because if none from both previous checks won't return anything, this function will return the last case by default